src/Cylinder.java

```diff
@@ -0,0 +1,37 @@
+public class Cylinder {
+    double height;
+    double radius;
+
+    // parameterized constructor
+    public Cylinder(double height, double radius) {
+        this.height = height;
+        this.radius = radius;
+    }
+
+    public double baseArea() {
+        try {
+            return (Math.PI * radius * radius); // return Base Area of shape Cylinder
+        } catch (ArithmeticException e) {
+            e.printStackTrace();
+            return 0.0;
+        }
+    }
+
+    public double volume() {
+        try {
+            return (Math.PI * radius * radius * height); //return Volume of shape Cylinder
+        } catch (ArithmeticException e) {
+            e.printStackTrace();
+            return 0.0;
+        }
+    }
+
+    public double surfaceArea() {
+        try {
+            return ((2 * 22 * radius) / 7) * (radius + height); //return Surface Area of shape Cylinder
```

Comment:
This is also good but In the above, you are using Math.PI but here you are using the value so I think you can use the same in all places.

Reply:
Got it. I will keep this in mind.